ZWaveLib/CommandClasses/ThermostatMode.cs

```diff
@@ -69,5 +69,18 @@ public static ZWaveMessage Set(ZWaveNode node, Value mode)
                 (byte)mode
             });
         }
+
+        public static ZWaveMessage Set(ZWaveNode node, int channel, Value mode) {
+			return node.SendDataRequest(new byte[] {
```

Comment:
Please, fix indentation.

Reply:
Sorry some tabs got included instead of spaces.